src/peergos/shared/user/SocialState.java

```diff
@@ -4,17 +4,30 @@
 import peergos.shared.user.fs.*;
 
 import java.util.*;
+import java.util.stream.*;
 
 @JsType
 public class SocialState {
 
     public final List<FollowRequest> pending;
     public final Map<String, FileTreeNode> followerRoots;
     public final Set<FileTreeNode> followingRoots;
+    public final Map<String, FileTreeNode> pendingFollowRequestsToOthers;
```

Comment:
is the ToOthers specifying anything particular or could  is just be named ``pendingFollowRequests``?